integration-tests/contract-manager-that-calls-failwith-contract.spec.ts

```diff
@@ -7,13 +7,13 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
   const test = require('jest-retries');
 
-  describe(`Test contract that calls 2nd contract that FAILs: ${rpc}`, () => {
+  describe(`Test contract origination of a contract that calls 2nd contract that FAILs through contract api: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    test('Catches a Failwith from a contract called via a manager contract', 2,  async (done: () => void) => {
+    test('Verify that contract.originate for a contract with code FAILWITH that links to another contract will fail', 2,  async (done: () => void) => {
```

Comment:
By reading the description it seems that it is the origination operation that fails, but in fact, it is the method call that fails (called via a manager contract)